libs/ui/src/components/search-panel/search-panel.tsx

```diff
@@ -41,15 +43,17 @@ export const SearchPanel: React.FC<Props> = ({
   isEmptyList,
   setSearchValue,
   selectedItemName,
-  onSearchClose,
   isSearchInitiallyOpened = false,
   onPressAddIcon,
-  onPressEditIcon,
   onPressActivityIcon,
-  style,
-  emptyIconStyle
+  setIsShowManageTokens,
+  emptyIconStyle,
+  isShowManageTokensIcon = false,
+  style
 }) => {
   const [isShowSearchField, setIsShowSearchField] = useState(isSearchInitiallyOpened);
+  const routeIndex = useNavigationState(state => state.index);
```

Comment:
maybe better to use here 
`import { useIsFocused } from '@react-navigation/native';`
`const isFocused = useIsFocused();`